src/main.rs

```diff
@@ -63,19 +71,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Comments
     println!("\n-------- COMMENTS \u{1F4AC} --------\n");
     let comments = &data["Comment"]["Comments"]["CommentsList"];
-    let comments_len = comments
-        .as_array()
-        .map(|array| array.len())
-        .unwrap_or(0);
+    let comments_len = comments.as_array().map(|array| array.len()).unwrap_or(0);
```

Comment:
```suggestion
    let comments_len = comments.as_array().map(|array| array.len()).unwrap_or(0);
```
I found the old syntax more readable. Can you adapt this please ?